lib/srv/termhandlers.go

```diff
@@ -94,6 +100,23 @@ func (t *TermHandlers) HandlePTYReq(ch ssh.Channel, req *ssh.Request, ctx *Serve
 		ctx.Errorf("Unable to update session: %v", err)
 	}
 
+	// update the user accounting database about the new tty
+	remoteStringIP, _, _ := net.SplitHostPort(ctx.ConnectionContext.ServerConn.RemoteAddr().String())
+	remoteIP := net.ParseIP(remoteStringIP)
+	hostname, err := os.Hostname()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	tty := term.TTY()
+	ttyName, err := utils.TtyName(tty)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	err = utils.InteractiveSessionOpened(ctx.Identity.Login, hostname, remoteIP, *ttyName)
```

Comment:
Not sure if this is the right place to call this function a lot of this code resides in `reexec.go`. Any reason why this doesn't reside there?

Reply:
Only reason is me not having enough understanding of how everything works when I wrote that code. Pushing some commits moving the hook-in to reexec.go in a sec.